### PR TITLE
feat(index): customize with passed args instead of requiring script edits

### PR DESCRIPTION
**Pull request by smb2796**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -35,18 +50,18 @@ if (argv.gasprice < 1 || argv.gasprice > 1000) throw "--gasprice must be between
...
+    priceFeedIdentifier: padRight(utf8ToHex(argv.priceFeedIdentifier.toString()), 64), // Price identifier to use.
+    syntheticName: argv.syntheticName, // Long name.
+    syntheticSymbol: argv.syntheticSymbol, // Short name.
     collateralRequirement: { rawValue: toWei("1.25") }, // 125% collateral req.
     disputeBondPct: { rawValue: toWei("0.1") }, // 10% dispute bond.
-    sponsorDisputeRewardPct: { rawValue: toWei("0.05") }, // 5% reward for sponsors who are disputed invalidly
+    sponsorDisputeRewardPct: { rawValue: toWei("0.05") }, // 5% reward for sponsors who are disputed invalidly.
     disputerDisputeRewardPct: { rawValue: toWei("0.2") }, // 20% reward for correct disputes.
-    minSponsorTokens: { rawValue: toWei("100") }, // Min sponsor position size of 100 synthetic tokens.
+    minSponsorTokens: { rawValue: toWei(argv.minSponsorTokens.toString()) }, // Minimum sponsor position size.
```
Comment (mrice32) [inline index.js:64]: Do any of these need `.toString()`? I think telling minimist to interpret them as strings guarantees that they're either strings or undefined/null.

**Review by mrice32 — APPROVED** [review]
LGTM!

File: index.js (hunk 1 above)
Comment (smb2796) [reply index.js:64]: I was getting an `invalid BigNumber value` error without `.toString()`, even when passing the args in as strings. Open to other suggestions on how to solve this.

**Review by pemulis — COMMENTED** [review]
I like it! LGTM

**Pull request merged**